examples/fungible/README.md

```diff
@@ -0,0 +1,270 @@
+<!-- cargo-rdme start -->
+
+# Fungible Token Example Application
+
+This example application implements fungible tokens, which demonstrate cross-chain messages. This
+application can be built and have its bytecode published on a Linera chain. The published bytecode
+can then be used to create multiple application instances, where each instance represents a
+different fungible token.
+
+# How It Works
+
+Individual chains have a set of accounts, where each account has an owner and a balance. The same
+owner can have accounts on multiple chains, with a different balance on each chain. This means that
+an account's balance is sharded across one or more chains.
+
+There are two operations: `Transfer` and `Claim`. `Transfer` sends tokens from an account on the
+chain where the operation is executed, while `Claim` sends a message from the current chain to
+another chain in order to transfer tokens from that remote chain.
+
+Tokens can be transferred from an account to different destinations, such as transfering to:
+
+- other accounts on the same chain
+- the same account on another chain
+- other accounts on other chains
+- sessions so that other applications to use some tokens
+
+# Usage
+
+## Setting Up
+
+The WebAssembly binaries for the bytecode can be built and published using [steps from the
+book](https://linera-io.github.io/linera-documentation/getting_started/first_app.html), summarized
+below.
+
+First setup a local network with two wallets, and keep it running in a separate terminal:
+
+```bash
+./scripts/run_local.sh
+```
+
+Compile the `fungible` application WebAssemlby binaries, and publish them as an application
+bytecode:
+
+```bash
+export LINERA_WALLET="$(realpath target/debug/wallet.json)"
+export LINERA_STORAGE="rocksdb:$(dirname "$LINERA_WALLET")/linera.db"
+export LINERA_WALLET_2="$(realpath target/debug/wallet_2.json)"
+export LINERA_STORAGE_2="rocksdb:$(dirname "$LINERA_WALLET_2")/linera_2.db"
+
+cd examples/fungible && cargo build --release && cd ..
+linera --wallet "$LINERA_WALLET" --storage "$LINERA_STORAGE" publish-bytecode \
+examples/target/wasm32-unknown-unknown/release/fungible_{contract,service}.wasm
+```
+
+This will output the new bytecode ID, e.g.:
+
+```rust
```

Comment:
(How is this passing the tests in CI??)

```suggestion
```ignore
```

Reply:
Oh wow, it was `ignore` in `lib.rs` but not in the `README.md`. I think `cargo rdme` replaces `ignore` with `rust`.

Reply:
Ah, I see! :+1: